tests/data/sample_ci_benchmark/benchmark.yml

```diff
@@ -0,0 +1,92 @@
+name: Benchmark GTSFM on select datasets using SIFT and Deep front-ends
```

Comment:
is this file duplicated?

Reply:
it is. i'm including it in `tests/data/sample_ci_benchmark/benchmark.yml` so that we even if we change the benchmark datasets in the future, the unit test will still be valid

Reply:
is it not better to update the unit tests every time we change the benchmarks? It might not need update if we only removing or adding datasets, but would need update if we change the format or something.

Reply:
I think it will be too onerous to have to update the unit tests every time we update a benchmark.

In my ideal world, the unit test is written one time, and passes, we don't have to look at it again, we can focus on other things instead.